.github/actions/send-slack-notification/action.yml

```diff
@@ -50,16 +50,18 @@ runs:
       run: |
         case $EVENT_NAME in
           pull_request)
-            BLOCK_MESSAGE=${BLOCK_MESSAGE:-${{ github.event.pull_request.title }}}
+            BLOCK_MESSAGE="${BLOCK_MESSAGE:-${{ github.event.pull_request.title }}}"
             ;;
           issues)
-            BLOCK_MESSAGE=${BLOCK_MESSAGE:-${{ github.event.issue.body }}}
+            BLOCK_MESSAGE="${BLOCK_MESSAGE:-${{ github.event.issue.body }}}"
             ;;
           *)
-            BLOCK_MESSAGE=${BLOCK_MESSAGE:-${{ github.event.head_commit.message }}}
+            BLOCK_MESSAGE="${BLOCK_MESSAGE:-${{ github.event.head_commit.message }}}"
             ;;
         esac
-        echo "result=${BLOCK_MESSAGE}" >> $GITHUB_OUTPUT
+        echo 'result<<EOF' >> $GITHUB_OUTPUT
+        echo $BLOCK_MESSAGE >> $GITHUB_OUTPUT
+        echo 'EOF' >> $GITHUB_OUTPUT
```

Comment:
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#example-2

still not sure how `variable<<` could work but it did